core/src/main/java/org/renjin/primitives/packaging/Namespace.java

```diff
@@ -96,7 +94,7 @@ public String getCompatibleName() {
   }
 
   public SEXP getEntry(Symbol entry) {
-    SEXP value = namespaceEnvironment.getVariable(entry);
+    SEXP value = namespaceEnvironment.getVariableUnsafe(entry);
```

Comment:
So we are assuming that the namespace environment does not contain active bindings. Do we enforce this? Are we sure that packages cannot define active bindings?